src/Umbraco.Core/Cache/Refreshers/Implement/UserCacheRefresher.cs

```diff
@@ -36,6 +36,21 @@ public override void Refresh(int id)
         base.Refresh(id);
     }
 
+    public override void Refresh(Guid id)
```

Comment:
See if `Remove(int id)` can be removed or somehow turned into less duplicate code.